Library/Formula/xmlto.rb

```diff
@@ -15,6 +15,8 @@ class Xmlto < Formula
   depends_on 'docbook'
   depends_on 'docbook-xsl'
   depends_on 'gnu-getopt'
+  # Doesn't strictly depend on GNU getopt, but OS X system getopt(1)
+  # does not suport longopts in the optstring, so use GNU getopt.
 
```

Comment:
I've fixed it up here but just for future reference comments about lines should generally be above rather than below the line.